src/OrchardCore/OrchardCore.Email.Core/Services/SmtpService.cs

```diff
@@ -71,7 +72,14 @@ public async Task<SmtpResult> SendAsync(MailMessage message)
                     message.From = senderAddress;
                 }
 
-                var mimeMessage = FromMailMessage(message);
+                var errors = new List<LocalizedString>();
+
+                var mimeMessage = FromMailMessage(message, errors);
```

Comment:
Not sure if errors should be part of `FromMailMessage` method? may be just create IsValid method separate from FromMailMessage method

Reply:
@ns8482e this is how the class was designed. All we did here is create more checks to ensure we report proper error instead of throwing exception